setup.py

```diff
@@ -42,7 +42,6 @@
         "Operating System :: POSIX :: Linux",
         "Operating System :: MacOS :: MacOS X",
         "Operating System :: POSIX :: SunOS/Solaris",
-        "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
```

Comment:
Add a line for 3.9?

Reply:
Good catch @JacobCallahan. I'm not sure why there's no line _here_ but 3.9 is listed in `test.yml`, but 3.9 is already listed [here](https://github.com/tldr-pages/tldr-python-client/blob/f24f81fb01d2b1bb69840bc6f98921fcbd700033/setup.py#L49) 🤔

@nebnola maybe your fork is out of date?

Reply:
I'm not sure what you mean. 3.9 continues to be listed, there is no change there. See [here](https://github.com/nebnola/tldr-python-client/blob/14d458e38c7f52725e92237f8dc6ccddb9a0bbb9/setup.py#L48)

Reply:
Oh! I didn't notice that I had to expand the hidden lines in the GH UI—the 3.9 line is the first line that's hidden :P